src/lib/branchpredictors/GenericPredictor.cc

```diff
@@ -22,35 +22,41 @@ GenericPredictor::GenericPredictor(ryml::ConstNodeRef config)
   // Create branch prediction structures
   btb_ =
       std::vector<std::pair<uint8_t, uint64_t>>(1 << btbBits_, {satCntVal, 0});
```

Comment:
Given the BTB-Tag-Bits can be up to 64, shouldn't the `1` here have the `ull` added?